packages/datastore/src/types.ts

```diff
@@ -434,13 +434,62 @@ export type DataStoreConfig = {
 		maxRecordsToSync?: number; // merge
 		syncPageSize?: number;
 		fullSyncInterval?: number;
+		syncExpressions?: SyncExpression<any>[];
```

Comment:
I think this can be changed to:

```suggestion
		syncExpressions?: SyncExpression<PersistentModel>[];
```